lang/lang_en_fr.txt

```diff
@@ -2,6 +2,14 @@
 "[%.7s]Live adj. Z\x0avalue set, continue\x0aor start from zero?\x0a%cContinue%cReset"
 "[%.7s]Ajust. du Z\x0aValeur enreg, contin\x0aou depart a zero?\x0a%cContinuer%cReset"
 
+#MSG_03_OR_OLDER c=18
+" 0.3 or older"
+" 0.3 ou plus vieux"
+
+#MSG_04_OR_NEWER c=18
+" 0.4 or newer"
+"0.4 ou plus recent"
```

Comment:
missing space - ok, the text needs 18 chars. The space may be added if we find a shorter translation

Reply:
I would prefer NOT to have a starting space as other Support messages.
Only one that are
Firmware:
Fil. sensor v.:

We could reuse "Fil. sensor" message here ... not sure that the `v.` is really needed.